frontend/model/captureLogs.js

```diff
@@ -0,0 +1,186 @@
+import sbp from '~/shared/sbp.js'
+import { CAPTURED_LOGS, SET_APP_LOGS_FILTER } from '~/frontend/utils/events.js'
+
+/*
+  giConsole/count - total logs stored
+  giConsole/lastEntry - latest log hash.
+  giConsole/markers - an array of markers at every 1000th entry.
+*/
+const ENTRIES_LIMIT = 4000
+const ENTRIES_MARKER = 1000
+
+let isCapturing = true
+let isConsoleOveride = false
+let lastEntry = null
+let entriesCount = null
+let appLogsFilter = []
```

Comment:
Could you please add a comment above these explaining that they are initialized correctly in `captureLogsStart()` and not here?